app/config/services/admin.xml

```diff
@@ -577,6 +561,14 @@
 
         <service autowire="true" public="false" id="AppBundle\Form\DataTransformer\CommitteeTransformer"/>
 
+        <service id="app.admin.amp_synchronised_extension" class="AppBundle\Admin\Extension\AmpSynchronisedAdminExtension">
+            <argument type="service" id="AppBundle\Form\EventListener\AmpSynchronisedListener" />
+
+            <tag name="sonata.admin.extension" target="app.admin.article" />
```

Comment:
Clarfication admin is missing no ?

Reply:
this admin is not declared at all. i guess we have to remove it completely :)

Reply:
@guillaumedenhez confirmed: we should remove the `ClarificationAdmin` code for now as it is useless.